shared/src/packages/users/libs/types/user-business-sign-up-request-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { type BusinessEntityT } from '~/packages/business/business.js';
```

Comment:
Why BusinessEntityT is exported, why not BusinessEntity 🤔

Reply:
Used at the same maner as we use UserEntityT, may be to visualy destinguish type from class